tools/review_tracker.py

```
@@ -0,0 +1,389 @@
+        # Total files
+        cursor.execute("SELECT COUNT(*) FROM reviews")
+        total = cursor.fetchone()[0]
+
+        # Never reviewed
+        cursor.execute("SELECT COUNT(*) FROM reviews WHERE last_reviewed IS NULL")
+        never_reviewed = cursor.fetchone()[0]
+
+        # Reviewed
+        reviewed = total - never_reviewed
+
+        print(f"Review Statistics:")
```

Ruff (0.13.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)